app/controllers/completed_applications_controller.rb

```diff
@@ -3,7 +3,9 @@ class CompletedApplicationsController < DashboardController
                 :present_crime_application, only: [:show]
 
   def index
-    @applications = applications_from_datastore.page(params[:page])
+    @applications = Datastore::GetApplications.new(
+      **pagination_params
```

Comment:
I suggest to keep the filter params (status, but soon may be others) and the pagination params separate, it is confusing otherwise as not all is pagination.